apps/extension/src/setup/index.ts

```diff
@@ -0,0 +1,3 @@
+import Setup from "../Setup";
+
+export default Setup();
```

Comment:
I got problem here as we have 2 directories that only differ in casing
```bash
/src/setup
/src/Setup
```
also I have compiler warning in `/src/setup/index.ts` that I expect to be due to this: `'default' implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer.ts(7022)`

I see all the files in those 2 directories listed in one folder in VSCode, Sublime Text and in Terminal. So I expect this to be system wide on MacOS. I think it would be better to place all files truly in the same directory or have name that do not only differ in casing.

Reply:
I removed `src/setup`, and the build directly links to `src/Setup`. I did the same for the `popup` build pipeline, which was importing a function that renders the `src/App` React app, but this is not needed either.